src/main/java/com/imsweb/geocoder/PointInPolygonOutput.java

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright (C) 2019 Information Management Services, Inc.
+ */
+package com.imsweb.geocoder;
+
+import java.io.IOException;
+
+import okhttp3.ResponseBody;
+import retrofit2.Call;
+
+import com.imsweb.geocoder.exception.BadRequestException;
+
+public class PointInPolygonOutput {
+
+    public static String CENSUS_CODE_1990 = "NineteenNinety";
+    public static String CENSUS_CODE_2000 = "TwoThousand";
+    public static String CENSUS_CODE_2010 = "TwoThousandTen";
+    public static String CENSUS_CODE_ALL = "AllAvailable";
```

Comment:
Are these actually used?

Reply:
No, but they should have been. The aggie API returns those codes to indicate the year corresponding to the data being returned. I'm not sure what "AllAvailable" is supposed to mean, since it doesn't seem to support returning multiple results.

In any case, I think the value returned should be translated from those codes into numbers; I guess I just forgot!